pkg/clusteragent/clusterchecks/metrics.go

```diff
@@ -23,6 +23,9 @@ var (
 	dispatchedConfigs = telemetry.NewGaugeWithOpts("cluster_checks", "configs_dispatched",
 		[]string{"node", le.JoinLeaderLabel}, "Number of check configurations dispatched, by node.",
 		telemetry.Options{NoDoubleUnderscoreSep: true})
+	dispatchedEndpoints = telemetry.NewGaugeWithOpts("endpoint_checks", "configs_dispatched",
```

Comment:
Should we also delete the gauge in the `expire_nodes` func like we do with the other metrics here https://github.com/DataDog/datadog-agent/blob/main/pkg/clusteragent/clusterchecks/dispatcher_nodes.go#L141 ?

Reply:
Endpoint checks are not dynamically assigned (unlike cluster checks). An endpoint check can only be assigned to a specific node. In other words, the pool of cluster check runners would have no impact on endpoint checks dispatching.